app/controllers/projects_controller.rb

```diff
@@ -37,6 +36,12 @@ def archive
     head :no_content
   end
 
+  def archive_user
+    worker = @projects.workers.find_by(user_id: params[:user_id])
```

Comment:
why `@projects`? If we can call `workers` on it then this is single instance of the model, right?